components/BlazerCarousel.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from "react";
+import { ChevronRight, ChevronLeft } from "lucide-react";
+import { BLAZER_IMAGES,FEATURED_PRODUCTS } from "@/utils/constants.js";
+import Image from "next/image";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+import Slider from "react-slick";
+import Marquee from "react-fast-marquee";
+
+
+const BlazerCarousel = () => {
+  const PrevArrow = ({ currentSlide, slideCount, ...props }) => {
+    const { onClick } = props;
+
+    return (
+      <div
+        {...props}
+        className="bg-white h-[30px] z-20 w-[30px] text-black rounded-full cursor-pointer top-[40%] left-10 absolute"
+        style={{display:"flex",alignItems:"center",justifyContent:"center"}}
+        onClick={onClick}
+      >
+      <ChevronLeft className="size-4" />
+      </div>
+    );
+  };
+  const NextArrow = ({ currentSlide, slideCount, ...props }) => {
+    const { onClick } = props;
+    return (
+      <div
+        {...props}
+        className="bg-white h-[30px] w-[30px] text-black rounded-full cursor-pointer top-[40%] right-10 absolute"
+        style={{display:"flex",alignItems:"center",justifyContent:"center"}}
+        onClick={onClick}
+      >
+        <ChevronRight size={20} />
+      </div>
+    );
+  };
```

Comment:
Create a separate component for PrevArrow and NextArrow

Reply:
We can avoid css repetition by creating a single component called `CarouselArrow` and pass the icon and onclick handler through props.